src/software/thunderscope/replay/replay_controls.py

```diff
@@ -125,7 +125,7 @@ def __on_replay_slider_pressed(self):
         self.player.pause()
         self.slider_pressed = True
 
-    def __on_replay_slider_value_changed(self, event):
+    def __on_replay_slider_value_changed(self: "ReplayControls", event) -> None:
```

Comment:
I think the `valueChanged` signal actually has no event [(the arg is just the new slider value)](https://doc.qt.io/qt-6/qabstractslider.html#valueChanged). So we should change this
  ```suggestion
    def __on_replay_slider_value_changed(self, value: int) -> None:
```